federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -63,6 +63,7 @@ const (
 	// other cluster.
 	firstClusterAnnotation = "ingress.federation.kubernetes.io/first-cluster"
 	ControllerName         = "ingresses"
+	UserAgentName          = "Federation-Ingresses-Controller"
```

Comment:
I see sync controller uses user agent of the form "ingress-controller". We should definitely add "federation" there.
Please update https://github.com/NickrenREN/kubernetes/blob/8862669061aa4e5aaf8284da746ad6f69f765133/federation/pkg/federation-controller/sync/controller.go#L91.
cc @marun @perotinus